compiler/modules/front.syntax/parseTypeDesc.bal

```diff
@@ -233,6 +233,19 @@ function parseFunctionTypeDesc(Tokenizer tok, FunctionParam[]? namedParams = ())
                 params.push(param);
                 check tok.advance();
             }
+            "..." => {
+                check tok.advance();
+                Position namePos = tok.currentStartPos();
+                string name = check tok.expectIdentifier();
```

Comment:
You don't have to have an identifier here. See

https://ballerina.io/spec/lang/2022R1/#rest-param

The param name is optional.

Reply:
fixed